src/AccessibilityInsights.SharedUx/Resources/Styles.xaml

```diff
@@ -886,6 +886,11 @@
                             <Setter TargetName="gdTab" Property="Background" Value="{DynamicResource ResourceKey=PrimaryBGBrush}"/>
                             <Setter TargetName="gdTab" Property="Background" Value="{DynamicResource ResourceKey=SelectedBrush}"/>
                         </MultiTrigger>
+                        <Trigger Property="IsFocused" Value="True">
+                            <Setter TargetName="Overlay" Property="BorderThickness" Value="3"/>
```

Comment:
Since the contrast issue only showed up in high contrast modes, you could use `BtnBrdrThickness` from Brushes.xaml (or follow a similar pattern) for the thickness here. This would avoid the "jumping" effect in non-HC modes. Might be more complicated than it's worth, though

Reply:
I tried this, but there was still a jump, I think due to the margin still being set. I'm going to leave this as is for now, and we have the option to revisit at another time. I made one change, though, which was to use SystemColors.WindowColor as the border, so we also get a contrasting border (as opposed to just a thicker border) in HC 1 and HC 2. I've updated the GIFs accordingly